### PR TITLE
Step3

### PR DESCRIPTION
**Pull request by utaash88**
## What

step3を完了させました

## CHECKS :warning:

Please make sure you are aware of the following.

- [ ] **The changes in this PR doesn't have private information

### CODE REVIEW

**Comment by mshibuya** [comment]
This implementation seems to be broken in several locations.
Please make sure that all features work as intended by using `curl`, before opening a PR!

File: python/main.py [hunk 1]
```diff
@@ -18,21 +23,54 @@
...
+    
+
+    return {"message": f"item received: {name},{category},{image}"}
+
+@app.get("/items")
+def get_items():
+    return json_load
 
+@app.get("/items/{items_id}")
+def get_items(items_id:int):
```
Comment (rhomel) [inline python/main.py]: This function is missing a body. It also has the same name as the previous function (`get_items`)

Comment (utaash88) [reply python/main.py]: Thank you

File: python/main.py [hunk 2]
```diff
@@ -18,21 +22,71 @@
...
+    with open("items.json", "r") as f:
+        items = json.load(f)
+    return items
 
+@app.get("/items/{item_id}")
+def get_item_id(item_id: int):
+    with open("items.json", "r") as f:
+        data = json.load(f)
+    
+    if 0 <= item_id < len(data["items"]):
```
Comment (rhomel) [inline python/main.py:82]: nice!

File: python/main.py [hunk 3]
```diff
@@ -18,21 +22,71 @@
...
+    else:
+        raise HTTPException(status_code=404, detail="Item not found")
+    
 @app.get("/image/{image_name}")
 async def get_image(image_name):
-    # Create image path
+    #image path
     image = images / image_name
 
     if not image_name.endswith(".jpg"):
```
Comment (rhomel) [inline python/main.py:92]: You can also check if the file exists with [python os.path](https://docs.python.org/ja/3/library/os.path.html#os.path.exists)

File: go/go.sum [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+golang.org/x/sys v0.0.0-20211103235746-7861aae1554b h1:1VkfZQv42XQlA/jchYumAnv1UPo6RgF9rJFkTgZIxO4=
+golang.org/x/sys v0.0.0-20211103235746-7861aae1554b/go.mod h1:oPkhp1MJrh7nUepCBck5+mAzfO9JrbApNNgaTdGDITg=
+golang.org/x/text v0.3.7 h1:olpwvP2KacW1ZWvsR7uQhoyTYvKAupfQrRGBFM352Gk=
+golang.org/x/text v0.3.7/go.mod h1:u+2+/6zg+i71rQMx5EYifcz6MCKuco9NR6JIITiCfzQ=
+golang.org/x/time v0.0.0-20201208040808-7e3f01d25324 h1:Hir2P/De0WpUhtrKGGjvSb2YxUgyZ7EFOSLIcSSpiwE=
+golang.org/x/time v0.0.0-20201208040808-7e3f01d25324/go.mod h1:tRJNPiyCQ0inRvYxbN9jk5I+vvW/OXSQhTDSoE431IQ=
+gopkg.in/check.v1 v0.0.0-20161208181325-20d25e280405/go.mod h1:Co6ibVJAznAaIkqp8huTwlJQCZ016jof/cbN4VW5Yz0=
+gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c/go.mod h1:K4uyk7z7BCEPqu6E+C64Yfv1cQ7kz7rIZviUmN+EgEM=
+gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b h1:h8qDotaEPuJATrMmW04NCwg7v22aHH28wwpauUhK9Oo=
+gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b/go.mod h1:K4uyk7z7BCEPqu6E+C64Yfv1cQ7kz7rIZviUmN+EgEM=
```
Comment (rhomel) [inline go/go.sum:38]: BTW, you only have to implement it with python or Go. So if you want to continue with Python, you can ignore this file.

**Review by rhomel — APPROVED** [review]
nice job!

**Pull request merged**